pkg/lib/config/authenticator.go

```diff
@@ -58,34 +79,70 @@ var _ = Schema.Add("PasswordPolicyConfig", `
 		"minimum_guessable_level": { "type": "integer" },
 		"excluded_keywords": { "type": "array", "items": { "type": "string" } },
 		"history_size": { "type": "integer" },
-		"history_days": { "$ref": "#/$defs/DurationDays" }
+		"history_days": { "$ref": "#/$defs/DurationDays" },
+		"expiry": { "$ref": "#/$defs/PasswordExpiryConfig" }
```

Comment:
```yaml
password:
  # The original policy configuration.
  # I think password expiry should not be put in it.
  # The things in policy is more about password requirement.
  # If we can rename it, it should be called "requirement" instead.
  policy:
    min_length: 8
  # This force_change is for password.policy.
  force_change: true

  # The 3 cases can be turned on individually.
  # However, the precedence is deny_login > force_change > prompt_change.
  expiry: # "expiration" is American English, while "expiry" is British English. But we have been using "expiry" in the config. So let's stick with it.
    # In this case, the authflow will result in a dead end, with an error telling that the password is expired and the login is denied.
    deny_login:
      enabled: true
      since_last_update_in_days: 90
    # In this case, the authflow will enter the change_password step (The existing one). To proceed, the end-user must change the password.
    force_change:
      enabled: true
      since_last_update_in_days: 60
    # In this case, the authflow will enter the change_password step. But this step now supports taking an input to skip the password update.
    prompt_change:
      enabled: true
      since_last_update_in_days: 30
```

Reply:
- Rename `since_last_update` to `duration_since_last_update`.
- Copy the above config and paste it as comment around `PasswordExpiryConfig` for future reference.